packages/core/services/managers/polyline-manager.ts

```diff
@@ -1,39 +1,35 @@
 import { Injectable, NgZone } from '@angular/core';
 import { Observable, Observer } from 'rxjs';
 
-import { AgmPolyline, PathEvent } from '../../directives/polyline';
+import { AgmPolyline } from '../../directives/polyline';
 import { AgmPolylinePoint } from '../../directives/polyline-point';
-import { createMVCEventObservable } from '../../utils/mvcarray-utils';
+import { createMVCEventObservable, MVCEvent } from '../../utils/mvcarray-utils';
 import { GoogleMapsAPIWrapper } from '../google-maps-api-wrapper';
-import { IconSequence, LatLng, LatLngLiteral, MVCArray, Polyline } from '../google-maps-types';
-
-declare var google: any;
 
 @Injectable()
 export class PolylineManager {
-  private _polylines: Map<AgmPolyline, Promise<Polyline>> =
-      new Map<AgmPolyline, Promise<Polyline>>();
+  private _polylines: Map<AgmPolyline, Promise<google.maps.Polyline>> =
+      new Map<AgmPolyline, Promise<google.maps.Polyline>>();
 
   constructor(private _mapsWrapper: GoogleMapsAPIWrapper, private _zone: NgZone) {}
 
-  private static _convertPoints(line: AgmPolyline): Array<LatLngLiteral> {
+  private static _convertPoints(line: AgmPolyline): google.maps.LatLngLiteral[] {
     const path = line._getPoints().map((point: AgmPolylinePoint) => {
-      return {lat: point.latitude, lng: point.longitude} as LatLngLiteral;
+      return {lat: point.latitude, lng: point.longitude} as google.maps.LatLngLiteral;
     });
     return path;
   }
 
-  private static _convertPath(path: 'CIRCLE' | 'BACKWARD_CLOSED_ARROW' | 'BACKWARD_OPEN_ARROW' | 'FORWARD_CLOSED_ARROW' |
-  'FORWARD_CLOSED_ARROW' | string): number | string{
-    const symbolPath = google.maps.SymbolPath[path];
+  private static _convertPath(path: keyof typeof google.maps.SymbolPath | string): number | string {
```

Comment:
it should return `SymbolPath|string`, not `number|string`

Reply:
thanks, updated.